.travis.yml

```diff
@@ -10,9 +10,14 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
+before_install:
+  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
 env:
   - KOREBUILD_DNU_RESTORE_CORECLR=true KOREBUILD_TEST_DNXCORE=true
```

Comment:
@NTaylorMullen Razor repo's travis.yml has the following setting `MONO_THREADS_PER_CPU=2000`, should I be using it here too or does it not matter anymore as I guess `KOREBUILD_DNU_RESTORE_CORECLR` is to say that restore is done using coreclr dnx?